recipe/meta.yaml

```
@@ -1,41 +1,41 @@
-{% set version = "0.14.1" %}
+{% set version = "0.14.2" %}
 
 package:
   name: statsmodels
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/statsmodels/statsmodels-{{ version }}.tar.gz
-  sha256: 2260efdc1ef89f39c670a0bd8151b1d0843567781bcafec6cda0534eb47a94f6
+  sha256: 890550147ad3a81cda24f0ba1a5c4021adc1601080bd00e191ae7cd6feecd6ad
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py<37]
+  skip: true  # [py<39]
```

Unfortunately, PyPy is on its way out. There no reason to block things on this here.
```suggestion
  skip: true  # [py<39]
  skip: true  # [python_impl == "pypy"]
```